core/test/unit/server_helpers/excerpt_spec.js

```diff
@@ -31,34 +31,38 @@ describe('{{excerpt}} Helper', function () {
                 '< test > those<<< test >>> who mistake it &lt;for&gt; binary.',
             expected = 'There are 10 types of people in the world: those who understand trinary, those who ' +
                 'don\'t and those>> who mistake it &lt;for&gt; binary.',
+            expected2 = '<p>There are <br />10<br> types<br/> of people in the world: those who understand trinary</p>, those who don\'t',
             rendered = helpers.excerpt.call({html: html});
 
         should.exist(rendered);
-        rendered.string.should.equal(expected);
+        rendered.string.should.equal(expected || expected2);
```

Comment:
This will always pass because it's essentially saying, "if the variable `expected` exists, the rendered string should equal it. Otherwise, the rendered string should equal `expected2`. Because we know for sure that expected exists (it is defined above), the tests will never check if expected2 matches the rendered string.

This should instead be split out into several different tests. That way it can be checked correctly.